execute/tokendata/usdc/attestation_test.go

```diff
@@ -0,0 +1,204 @@
+package usdc
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	commonconfig "github.com/smartcontractkit/chainlink-common/pkg/config"
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-ccip/execute/exectypes"
+	"github.com/smartcontractkit/chainlink-ccip/pkg/reader"
+	"github.com/smartcontractkit/chainlink-ccip/pluginconfig"
+)
+
+func createHandler(t *testing.T, success []string, pending []string) http.HandlerFunc {
+	successes := make(map[string]string)
```

Comment:
This function could be optimized a bit.

- dedup "/v1/attestations/0x" + hash
- response := writeResponse(status="complete", attestation="...")